### PR TITLE
Add ESLint rules for consistent spacing after keywords and comments

### PR DESCRIPTION
**Pull request by elfenheart**
We should be more strict with code styles. Will add more rules as we go.

### CODE REVIEW

File: cdap-ui/app/cdap/components/AdminConfigurePane/index.js [hunk 1]
```diff
@@ -24,12 +24,11 @@ var configButtons = [
...
 ];
 
-export default function AdminConfigurePane({ openNamespaceWizard }){
+export default function AdminConfigurePane({ openNamespaceWizard }) {
 
   var buttons = [];
-  for(var i = 0 ; i < configButtons.length; i++){
-
-    if(i === 0){
+  for (var i = 0 ; i < configButtons.length; i++) {
```
Comment (wolf31o2) [inline cdap-ui/app/cdap/components/AdminConfigurePane/index.js]: Space before second semicolon?

**Review by wolf31o2 — APPROVED** [review]
One question about spacing inside for loops, but nothing blocking. 👍 LGTM

File: cdap-ui/app/cdap/components/AdminConfigurePane/index.js (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/AdminConfigurePane/index.js]: Nice catch! Added rule to enforce spacing around `;`

**Comment by wolf31o2** [comment]
👍

**Pull request merged**